server/routerlicious/packages/routerlicious-base/src/nexus/README.md

```diff
@@ -0,0 +1,7 @@
+# Nexus
+
+Nexus is a microservice that handles the Socket.IO that used to belong to Alfred. This means Nexus now handles the operation stream (websocket connections) part of routerlicious while Alfred keeps serving the HTTP document APIs.
+
+In order for the Nexus routing to work correctly, the client must have the Discovery feature enabled. Alfred will then return the deltaStreamURL pointing to a Nexus URL.
```

Comment:
Targeting this bit but general question about the change: what kind of impact do we expect to existing workflows? E.g. The fact that this is exposed on a separate port in the docker compose setup makes me think our test drivers might not work in their current form (I believe there's some hardcoded logic that relies on ports 3000/3001 for everything alfred and everything historian, respecitvely). Also trying to figure out how this might impact tinylicious; at a glance I think that one is not impacted but I'm not entirely sure.

Reply:
From my understanding, utilizing Nexus will require using the Discovery functionality in the Routerlicious driver because that exposes the DeltaStreamUrl added a while back in #11428 by @evaliyev 

It shouldn't impact Tinylicious at all, since by definition Tinylicious is only 1 service.

Reply:
Also, this PR specifically does not actually disable Alfred's socket.io setup, so it will have no impact there. With this current setup in this PR, while I write this comment, nothing will change for drivers that use hardcoded ports.

Reply:
Hi Alex, will take a look at this, but existing workflows should be untouched. To be perfectly clear, in this PR Alfred is being kept as is, handling both HTTP and Websockets and we are only introducing an opt-in for using Nexus. Here, we are only adding the new service and in a later PR, we will remove the Websocket part from Alfred. In that future PR we can make sure to take care of those hardcoded bits.